app/controllers/projects_controller.rb

```diff
@@ -101,17 +100,29 @@ def set_project
     @project = Project.friendly.find(params[:id])
   end
 
-  def query_projects(status)
-    status ||= 'active'
-    @projects = Project.where(status: status)
-                       .order('last_github_update DESC NULLS LAST')
-                       .order('commit_count DESC NULLS LAST')
-                       .includes(:user)
+  def query_projects_alpha
+    status = params[:status]
+    @projects = if status == 'pending'
+                  Project.where(status: %w(pending)).order('title ASC')
```

Comment:
Can we try seeing if status: status works here instead of this %w(pending) thing?

Reply:
Fixed, and refactored duplicate code in query_projects.